src/setup.h

```diff
@@ -15,7 +17,7 @@ void
 uzbl_events_free ();
 
 void
-uzbl_gui_init ();
+uzbl_gui_init (WebKitWebContext *context);
```

Comment:
Yeah. gui would also work. The only difference would be that the webcontext is created a bit later (uzbl_gui_init) is the last to be called. I'll test that and see if it still works :)